files/ohpc_install.yml

```diff
@@ -189,9 +210,13 @@
                         compute_regex: ${compute_regex}
                         compute_prefix: ${compute_prefix}
                         compute_nodes:
-                        - { num: 1, c_name: "${cnode01}", c_ip: "${cnode01ip}", c_mac: "${cnode01mac}", c_bmc: "10.41.1.0"}
-                        - { num: 2, c_name: "${cnode02}", c_ip: "${cnode02ip}", c_mac: "${cnode02mac}", c_bmc: "10.41.1.0"}
-                        - { num: 3, c_name: "${cnode03}", c_ip: "${cnode03ip}", c_mac: "${cnode03mac}", c_bmc: "10.41.1.0"}
+                        $(p=0; for i in "${cnodes[@]}"; do ip=ip_${i}; mac=mac_${i}; let "p++"; echo -e "- { num: '${p}', c_name: '${i}', c_ip: '${!ip}', c_mac: '${!mac}', c_bmc: '10.41.1.0' }"; done)
+                        sms_ipoib_internal: ${master_ipoib_internal}
+                        cnodes_ipoib_internal: ${cnodes_ipoib_internal}
+                        sms_ipoib: ${ipoib_master}
+                        ipoib_netmask: ${ipoib_netmask}
+                        compute_ipoib:
+                        $(p=0; for i in "${cnodes[@]}"; do ip=ip_${i}; ipoib=ipoib_${i}; let "p++"; echo -e "- { num: '${p}', c_hostname: '${!ip}', c_ipoib: '${!ipoib}', c_cid: 'ff:00:00:00:00:00:02:00:00:02:c9:00:50:80:02:00:00:55:00:01' }"; done)
```

Comment:
What is this hash?

Reply:
This is a variable in the Infiniband configuration that is not used, it is the same value as the default one. I put it there mainly to not forget about it, if need for it ever arises.

Reply:
Better to leave it out, then. Let's carry that information in a ticket somewhere, in case we need it, but not in the code.

Reply:
Fair enough !